llvm/include/llvm/Object/COFFImportFile.h

```diff
@@ -135,6 +135,18 @@ struct COFFShortExport {
   }
 };
 
+/// Writes a COFF import library containing entries described by the Exports
+/// array.
+///
+/// For hybrid targets such as ARM64EC, additional native entry points can be
+/// exposed using the NativeExports parameter. When NativeExports is used, the
+/// output import library will expose these native ARM64 imports alongside the
+/// entries described in the Exports array. Such a library can be used for
+/// linking both ARM64EC and pure ARM64 objects, and the linker will pick only
+/// the exports relevant to the target platform.
+///
+/// For non-hybrid targets, the NativeExports parameter should not be used.
+/// Instead, pass std::nullopt or an empty array to this parameter.
```

Comment:
Actually, as most users won't need to care about this parameter at this point, and as it seems like this change can be disruptive to users, what do you think about moving the new parameter to the end of the parameter list, and adding a default `= std::nullopt` to it, to avoid forcing users to change right away?

I guess the usual LLVM policy is that we don't need to worry about downstream users, but IMO if we can avoid breaking them, that's of course even better.

Reply:
Sounds good to me, I did that in the new version, thanks! As I mentioned in #81426, I'm ultimately planning to use it in all in-tree callers. But indeed, I don't expect it to be interesting for majority of downstream users and making it optional seems like a good idea.

Reply:
Thanks, I think this seems reasonable. Do you agree @zmodem? It causes a bit of extra unnecessary back-and-forth here, but the change landed like <24h ago, so it's perhaps best to just back out of the unnecessary change.

Reply:
Yes, this lgtm.